.github/workflows/build-package.yml

```diff
@@ -36,14 +37,21 @@ jobs:
         TARGET_PLATFORM:
           - centos,8
           - centos,7
-          - debian,bullseye
-          - debian,buster
-          - debian,stretch
           - oraclelinux,8
           - oraclelinux,7
-          - ubuntu,focal
-          - ubuntu,bionic
-          - pgxn
+        POSTGRES_VERSION:
+          - 10
+          - 11
+          - 12
+          - 13
+          - 14
+        include:
+          - TARGET_PLATFORM: debian,bullseye
+          - TARGET_PLATFORM: debian,buster
+          - TARGET_PLATFORM: debian,stretch
+          - TARGET_PLATFORM: ubuntu,focal
+          - TARGET_PLATFORM: ubuntu,bionic
+          - TARGET_PLATFORM: pgxn
```

Comment:
What is the purpose of this include line? It is not so clear to me.

Reply:
I needed to seperate centos builds for each pg version since I got errors for centos 8. Before this development, 4 postgres docker image was build at the same process but it was very error prone and failed frequently. Now I seperated them and if one of them fail I have an opportunity to rerun just one docker image
Therefore I created a cartesian multiplication of PLATFORM and POSTGRES_VERSION for rpm based images.
However cartesian multiplication does not work for deb based images since there is only one image for each platform
include part of yml adds processes one by one without cartesian multipl.
For reference: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations